.buildkite/scripts/packaging/package-util.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+is_pr() {
```

Comment:
This function might be better in a generic utils tool?

Reply:
we will refactor all util and generic scripts at a later phase